src/components/Contacts/ContactPartnershipStatus/ContactLateStatusLabel/ContactLateStatusLabel.test.tsx

```diff
@@ -1,54 +1,121 @@
 import React from 'react';
 import { ThemeProvider } from '@mui/material/styles';
 import { render } from '@testing-library/react';
+import { Settings } from 'luxon';
 import theme from '../../../../theme';
-import {
-  ContactLateStatusEnum,
-  ContactLateStatusLabel,
-} from './ContactLateStatusLabel';
+import { ContactLateStatusLabel } from './ContactLateStatusLabel';
+
+Settings.now = () => new Date(2020, 1, 1).valueOf();
```

Comment:
Sorry, I think I confused you in my other comment. The `setup.ts` file runs for every test, so by default "now" is mocked to Jan 1, 2020. So this line isn't needed unless you wanted a different now timestamp.